openlibrary/templates/type/author/view.html

```diff
@@ -207,11 +211,11 @@ <h3>$:_('Links <span class="gray small sansserif">(outside Open Library)')</span
                         $ viaf = page.remote_ids.viaf
                         <li>VIAF: <a href="https://viaf.org/viaf/$viaf">$viaf</a></li>
                     $if page.remote_ids.wikidata:
-                        <li>Wikidata: <a href="https://tools.wmflabs.org/reasonator/?q=$page.remote_ids.wikidata">$page.remote_ids.wikidata</a></li>
+                        <li>Wikidata: <a itemprop="sameAs" href="https://tools.wmflabs.org/reasonator/?q=$page.remote_ids.wikidata">$page.remote_ids.wikidata</a></li>
                 $if page.wikipedia and page.wikipedia.startswith("http"):
-                    <li><a href="$page.wikipedia">Wikipedia</a></li>
+                    <li><a itemprop="sameAs" href="$page.wikipedia">Wikipedia</a></li>
                 $for link in page.links:
-                    <li><a href="$link.url">$link.title</a></li>
+                    <li><a itemprop="sameAs" href="$link.url">$link.title</a></li>
```

Comment:
I don't think we can say that any link on an author bio here will be `sameAs` , it could be a personal blog or a storefront or somethine esle. VIAF and ISNI links above should probably be `sameAs` though.

Reply:
will make these changes